src/components/ArticleComments.jsx

```diff
@@ -1,3 +1,49 @@
+import { useEffect,useState } from "react"
+import { getArticleById, getCommentsByArticleId } from "../API"
+import { useParams, Link } from "react-router-dom";
+import { CommentCard } from "./CommentCard";
+import { PostComment } from "./PostComment";
+
 export const ArticleComments = () => {
-    return <h1>Article Comments!</h1>
+
+    const [comments, setComments] = useState([])
+    const [article, setArticle] = useState({})
```

Comment:
If ArticleComments is a child of the Article component you could pass article down as a prop so wouldn't need to set the state again here